browser.jsx

```diff
@@ -1,9 +1,9 @@
 'use strict'
-var remote = require('remote')
-var Menu = remote.require('menu')
-var MenuItem = remote.require('menu-item')
-var clipboard = require('clipboard')
-var urllib = require('url')
+var remote = require('electron').remote
+var Menu = require('electron')
```

Comment:
In the previous version, Menu and MenuItem were remote-required:

```js
var Menu = remote.require('menu')
var MenuItem = remote.require('menu-item')
```

In this new version, they are not:

```js
var Menu = require('electron')
var MenuItem = require('electron')
var clipboard = require('electron')
var urllib = require('electron')
```

In fact, now that I look more closely, I see that that clipboard and urllib are also incorrectly imported.